### DISCUSSION TITLE
Opening up to contributors

### DISCUSSION THREAD

**Discussion by shivan-s** [mrdbourke/nutrify#73] [General]
Hi Dan,

What are your thoughts on contributors?

At the moment, the repository needs a cleaning up but I understand you also have lots of work to do.

At the moment, it would be difficult for contributors to step in and help out (unless I'm wrong).

You may have to dedicate time specifically to writing documentation to get contributors started as well as cleaning up the repo.

Hopefully, this is a constructive critique.

**Comment by mrdbourke** [comment]
At the moment you're right, the repo is a bit all over the place.

Organized chaos haha

Later on I'd be open for contributions.

But for now, while the project is still in it's infancy, I'm trying to figure out the workflow as we speak.

It's slowly approaching an alpha stage (it works for me but not 100% reproducible yet).

Once the engine is turning, I'll look into properly setting up contributions.

But for now, since I'm still changing/breaking things daily, I don't think it's ready for major contributions.

**Reply by shivan-s** [reply]
Awesome work, mate.
